src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -32,9 +34,11 @@ export default class GroupComparisonMutationsTab extends React.Component<
     }
 
     @action.bound
-    protected handleGeneChange(id: string | undefined) {
+    protected handleGeneChange(id: string) {
         this.geneTab = id;
-        this.props.store.setSelectedMutationMapperGene(id);
+        this.props.urlWrapper.updateURL({
```

Comment:
@gblaih i would think that this.geneTab is no longer needed now that the urlwrapper is source of truth?

Reply:
agreed, removed it.